conf.py

```diff
@@ -0,0 +1,283 @@
+import json
+import re
+import configparser
+import ast
+from pathlib import Path
+
+
+CONFIG_DIR = f'{Path.home()}/.config/spotirec'
+URI_RE = r'spotify:(artist|track):[a-zA-Z0-9]'
+
+
+def open_config() -> configparser.ConfigParser:
+    """
+    Open configuration file as object
+    :return: config object
+    """
+    try:
+        # Read config and assert size
+        c = configparser.ConfigParser()
+        c.read_file(open(f'{CONFIG_DIR}/spotirec.conf'))
+        assert len(c.keys()) > 0
+        return c
+    except (FileNotFoundError, AssertionError):
```

Comment:
Each key is a section header in the config file. The configparser module is unable to parse a config file if no headers are present. The program would simply exit if this was the case.

Besides, most, if not all, config entries are either complex or not possible to create yourself.

Reply:
Yes, okay. With the example I saw that makes more sense. If a user was so inclined as to edit a config themselves it would likely just be editing fields in an existing config.